.github/ISSUE_TEMPLATE/user-story-issue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: User Story
+about: This template provides a basic structure for user story issues.
+labels: Needs Triage
+---
+
+# Description
+As a ..., I want to ..., so I can ...
+
+<Link to Design Document or User Story>
+
+# Acceptance criteria
+
+- [ ] This is something that can be verified to show that this user story is satisfied.
+- [ ] Unit Tests are completed and code passes tests
+
+# Sprint Ready Checklist 
+- [ ] Acceptance criteria defined 
+- [ ] Team understands acceptance criteria 
+- [ ] Team has defined solution / steps to satisfy acceptance criteria 
+- [ ] Acceptance criteria is verifiable / testable 
```

Comment:
Consider bundling this with the first check (and adding it to the other templates)

`Verifiable / testable acceptance criteria are defined.`